src/test/java/com/ibm/eventstreams/connect/mqsource/JmsToKafkaHeaderConverterTest.java

```diff
@@ -0,0 +1,50 @@
+package com.ibm.eventstreams.connect.mqsource;
```

Comment:
Add copyright statement to match the other source files.

Reply:
done